### PR TITLE
Update hoisting.md

### PR DESCRIPTION
**Pull request by Stefan-Stojanovic**
add code identifier and back ticks to the 2nd example

### CODE REVIEW

**Comment by enki-hq** [comment]
### Insight previews:
- [ ] [enkidevs/curriculum/javascript/core/scope/hoisting.md](https://app.enkipro.com/#/public/preview-insight/enkidevs/curriculum/javascript/core/scope/hoisting.md?ref=ce212110710acca654cb3f92accd8cb61d705a3d)

File: javascript/core/scope/hoisting.md [hunk 1]
```diff
@@ -58,6 +58,7 @@ This default behaviour moves all variable declarations to the top of the current
 
 This is how the snippet is actually executed, in order:
 
+```javascript
     var a;
```
Comment (nem035) [inline javascript/core/scope/hoisting.md]: you should also move every line in this code to the left edge:

```suggestion
var a;
```

File: javascript/core/scope/hoisting.md [hunk 2]
```diff
@@ -59,14 +59,14 @@ This default behaviour moves all variable declarations to the top of the current
...
+function go(){
     var a;
-    a = 1;
-    function go(){
-        var a;
-        console.log(a);
-        a = 2;
-    }
-    go();
+    console.log(a);
```
Comment (nem035) [inline javascript/core/scope/hoisting.md]: This one has 4 spaces but should have 2, did you format the code?

File: javascript/core/scope/hoisting.md [hunk 3]
```diff
@@ -76,16 +77,16 @@ What will the following functions print? ???
 
 ```
```
Comment (kapnobatai137) [inline javascript/core/scope/hoisting.md]:
```suggestion
```javascript
```

**Pull request merged**